contracts/src/v0.8/dev/KeeperRegistryBase.sol

```diff
@@ -95,6 +109,7 @@ abstract contract KeeperRegistryBase is ConfirmedOwner, ExecutionPrevention, Ree
     uint96 minUpkeepSpend; // 1 full evm word
     uint32 maxPerformGas;
     uint32 nonce;
+    uint32 registryGasOverhead;
```

Comment:
can we have the registryGasOverhead as immutable variable to save storage read

Reply:
Done